app/javascript/components/pr-show.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div
+    class="card-pr__list"
+  >
+    <div class="card-pr-title">
+      Feed
+    </div>
+    <div class="card-pr-title__header">
+      <div class="card-pr-title__header-card">
+        <div class="card-pr-title__name">
+          {{ $t("message.prFeed.prName") }}
+        </div>
+        <div class="card-pr-title__type-1">
+          {{ $t("message.prFeed.prAuthor") }}
+        </div>
+        <div class="card-pr-title__type-1">
+          {{ $t("message.prFeed.prProject") }}
+        </div>
+        <div class="card-pr-title__type-1">
+          {{ $t("message.prFeed.prTime") }}
+        </div>
+        <div class="card-pr-title__type-1">
+          {{ $t("message.prFeed.prDate") }}
+        </div>
+      </div>
+      <div class="card-pr-title__header-button">
+        Likes
+      </div>
+    </div>
+    <div class="card-pr__orientation">
+      <div class="card-pr">
+        <a
+          class="card-pr__name"
+          target="_blank"
+          :href="pullRequest.html_url"
+        >
+          {{ pullRequest.title }}
+        </a>
+        <p
+          v-if="(pullRequest.owner_name)"
+          style="flex: 2;"
+        >
+          {{ pullRequest.owner_name }}
+        </p>
+        <p
+          v-else
+          style="flex: 2;"
+        >
+          {{ $t("message.prFeed.noName") }}
+        </p>
+        <p class="card-pr__project">
+          {{ pullRequest.repository_name }}
+        </p>
+        <p style="flex: 2;">
+          {{ prTime(pullRequest) }}
+        </p>
+        <p style="flex: 2;">
+          {{ prDate(pullRequest) }}
+        </p>
+      </div>
+      <div class="card-pr__card">
+        <p class="card-pr__circle">
+          {{ pullRequest.likes.total }}
+        </p>
+        <div v-if="!(pullRequest.currentUserLike)">
+          <button
+            class="card-pr__button"
+            @click="toggleLike(pullRequest)"
+          >
+            Like
+          </button>
+        </div>
+        <div v-else>
+          <button
+            class="card-pr__button"
+            @click="deleteLike(pullRequest)"
+          >
+            Dislike
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+
+import axios from 'axios';
+import moment from 'moment';
+
+export default {
+  props: {
+    pullRequest: {
+      type: Array,
+      required: true,
+    },
+    likesGiven: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  data() {
+    const liked = this.likesGiven.find((like) => like.likeable_id === pullRequest.id);
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'pullRequest' is not defined  no-undef